src/models/scenario.ts

```
@@ -0,0 +1,47 @@
+  @prop({ type: () => [String] }) problems?: string[];
+
+  @prop() isValid?: boolean;
+
+  // Static methods
+  static async findScenarioById(
+    this: ReturnModelType<typeof Scenario>,
+    _id: string
+  ): Promise<Scenario | null> {
+    try {
+      await connectToDatabase();
+      return await this.findOne({ _id }).lean();
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)